libs/ledger-live-common/src/families/ethereum/synchronisation.ts

```diff
@@ -677,3 +682,31 @@ function reconciliateSubAccounts(
 
   return subAccounts;
 }
+
+// keep the initial dates of the operations to avoid the dates are overwrited by the ones from the blockchain
+export function preserveInitialOperationsDate(
```

Comment:
We could unit test this method

Reply:
That's exactly what the test added in `synchronisation.test.ts` does, right?

Reply:
yes

Reply:
After discussing it together, we might need to split it in multiple test (one for each call) to cover every use case

cc @haammar-ledger @hzheng-ledger